code/modules/power/cable.dm

```diff
@@ -513,9 +513,8 @@ GLOBAL_LIST_INIT(cable_coil_recipes, list (new/datum/stack_recipe/cable_restrain
 		user.visible_message("<span class='suicide'>[user] is strangling [user.p_them()]self with [src]! It looks like [user.p_theyre()] trying to commit suicide.</span>")
 	return OXYLOSS
 
-/obj/item/stack/cable_coil/New(loc, length = MAXCOIL, paramcolor = null)
+/obj/item/stack/cable_coil/New(loc, new_amount, merge = TRUE, paramcolor = null)
```

Comment:
This seems like an unrelated change?

Reply:
As a co-author.
To make `/obj/item/stack/cable_coil/five` actually have... 5 cables in the stack, I got rid of the `lenght = MAXCOIL` because it was setting all stacks to contain 30 cables.

Removing just the `lenght` made `paramcolor` the second argument, and it became the `new_amount` in `/obj/item/stack/New(loc, new_amount, merge = TRUE)` , which caused runtimes, this is to prevent that. Unsurer if there are better solutions that do not incur larger scope creep.

Reply:
Ok, yeah I'm just worried about the arg position shift because there's a large amount of cable coil references in the code, and there could be an arg mismatch.

For now you could instead do:
```dm
/obj/item/stack/cable_coil/five

// Passes '5' to the parent as `new_amount`, so 5 coils are created.
/obj/item/stack/cable_coil/five/New(loc, new_amount = 5, merge = TRUE, paramcolor = null)
	..()
```

Reply:
Should be handled